src/components/Headers/UserProfileHeader/UserProfileHeader.module.scss

```diff
@@ -13,14 +13,20 @@
 }
 
 .withBackBackButton {
-  justify-content: space-between;
+  justify-content: left;
 }
 
-.actions{
+.actions {
   display: flex;
-  align-items: center;
-  justify-content: left;
+  align-items: flex-start;
+  justify-content: flex-start;
   flex-wrap: wrap;
   row-gap: 12px;
   column-gap: 10px;
 }
+
+@media (min-width: 768px) {
```

Comment:
When you use the `@include Mobile` mixin with `withBackButton` class, you won't need this rule anymore.